Rakefile

```diff
@@ -23,7 +23,7 @@ namespace :generate do
   desc "generate a fresh app with rspec installed"
   task :app do |_task|
     unless File.directory?('./tmp/example_app')
-      bindir = File.expand_path("bin")
+      bindir = File.expand_path("bin", './tmp/example_app')
```

Comment:
This changes what `bindir` is pointing to.

```ruby
bindir = File.expand_path("bin")
# => RSPEC_RAILS_REPO_DIR/bin
bindir = File.expand_path("bin", './tmp/example_app')
# => RSPEC_RAILS_REPO_DIR/tmp/example_app/bin
```

We want the original. This is specifically for handling installs which use `--binstubs`.